src/app/scripts/cac/pages/cac-pages-directions.js

```diff
@@ -40,10 +51,63 @@ CAC.Pages.Directions = (function ($, _, DirectionsList, Itinerary, Settings, Uti
             var itineraries = data.plan.itineraries;
             var params = data.requestParameters;
             var itinerary = new Itinerary(itineraries[itineraryIndex], itineraryIndex, params);
+            setMapItinerary(itinerary);
             directionsListControl.setItinerary(itinerary);
         }, function (error) {
             console.log('error: ', error);
         });
+
+        function loadMap() {
+            var mapOptions = {
```

Comment:
All map interactivity should be disabled here already. I don't think I missed anything?